tests/keras/applications/applications_test.py

```diff
@@ -235,6 +235,43 @@ def test_inceptionv3_variable_input_channels():
 
 
 @keras_test
+@pytest.mark.skipif((K.backend() == 'cntk'),
+                    reason='InceptionResNetV2 is not supported on CNTK')
+def test_inceptionresnetv2():
+    model = applications.InceptionResNetV2(weights=None)
+    assert model.output_shape == (None, 1000)
+
+
+@keras_test
+@pytest.mark.skipif((K.backend() == 'cntk'),
+                    reason='InceptionResNetV2 is not supported on CNTK')
+def test_inceptionresnetv2_notop():
+    model = applications.InceptionResNetV2(weights=None, include_top=False)
+    assert model.output_shape == (None, None, None, 1536)
+
+
+@keras_test
+@pytest.mark.skipif((K.backend() == 'cntk'),
+                    reason='InceptionResNetV2 is not supported on CNTK')
+def test_inceptionresnetv2_pooling():
+    model = applications.InceptionResNetV2(weights=None, include_top=False, pooling='avg')
+    assert model.output_shape == (None, 1536)
+
+
+@keras_test
+@pytest.mark.skipif((K.backend() == 'cntk'),
+                    reason='InceptionResNetV2 is not supported on CNTK')
+def test_inceptionresnetv2_variable_input_channels():
+    input_shape = (1, None, None) if K.image_data_format() == 'channels_first' else (None, None, 1)
```

Comment:
This test is only every run with the default image data format (channels_last), and in fact you are hard-coding this expectation (with the shape `(None, None, None, 1536)`). Please add a test for `channels_first` (set and later restore the global image_data_format).

Reply:
Sure.